### ISSUE TITLE
[web] implement Paint.strokeMiterLimit

### ISSUE THREAD

**Issue by yjbanov** [flutter/flutter#33605]
This is not implemented on the Web.

**Comment by shivance** [comment]
Is this issue still open for contribution? If yes, then I would like to take a crack at it!

**Comment by hterkelsen** [comment]
Yes, it is. From a quick glance it seems like it should be a matter of setting this https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/miterLimit when rendering paths to an HTML Canvas.

**Comment by yjbanov** [comment]
As the HTML renderer has been deprecated and removed from the default Flutter SDK, we're now closing all issues that affect only the HTML renderer. For more information, please see:

* https://github.com/flutter/flutter/issues/145954
* https://github.com/flutter/flutter/pull/164003
